xml/pmo/actives/simple.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2016-2019 Zerocracy
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to read
+the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+merge, publish, distribute, sublicense, and/or sell copies of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<users version="0.25" updated="2016-12-29T09:03:21.684Z">
+  <user id="g4s8">
+    <activated>2016-12-29T09:03:21.684Z</activated>
+  </user>
+  <user id="0crat">
+    <activated>2016-12-29T09:03:21.684Z</activated>
+  </user>
+</users>
```

Comment:
@g4s8 I'm not clear what this file is for?

Reply:
@victornoel this file is like a test resource, the build is running xsd schema against this file and checks that it's valid

Reply:
@g4s8 ah indeed, sorry I now realise the README explains that. So the work is finished for that task or do we need todos (or maybe you will be making multiple PRs?)?

Reply:
@victornoel yes, it's finished in this repo. It'll be continued in `farm`: https://github.com/zerocracy/farm/issues/2245

Reply:
@g4s8 I understand, thank you for the explanation :)